lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -210,8 +224,26 @@ def _get_saved_grade(self, subsection, course_structure, course):  # pylint: dis
                 )
                 subsection_grade = SubsectionGrade(subsection)
                 subsection_grade.load_from_data(model, course_structure, self._scores_client, self._submissions_scores)
+                log.warning(
+                    u"Persistent Grades: Loaded saved grade for course with id {0}, version {1}, subtree edited on {2},"
```

Comment:
Rationale for removing "saved" from these log statements is so it's easier to query for whenever a grade is actually saved.